.github/workflows/build.yml

```diff
@@ -57,28 +57,50 @@ jobs:
         os: [macos-latest, macos-12]
     runs-on: ${{ matrix.os }}
     timeout-minutes: 90
-    env:
-      CPATH: /opt/homebrew/include
-      LIBLARY_PATH: /opt/homebrew/lib
     steps:
-      - name: Instlall tools
+      - name: Install tools
         run: |
-          brew install gmp oniguruma automake re2c bison subversion gauche
+          brew install gmp oniguruma automake re2c bison gauche
+          # FIXME: Hardcode this
+          echo "HOMEBREW_PREFIX=/usr/local" >> $GITHUB_ENV
+          # FIXME: Detect gnump with pkg-config
+          echo "LDFLAGS=-L$HOMEBREW_PREFIX/lib" >> $GITHUB_ENV
+          echo "CFLAGS=-I$HOMEBREW_PREFIX/include" >> $GITHUB_ENV
+          echo "CXXFLAGS=-I$HOMEBREW_PREFIX/include" >> $GITHUB_ENV
+          # FIXME: Disable GC during CI for now
+          echo "GC_DONT_GC=1" >> $GITHUB_ENV
       - name: Install the latest stable Mosh
         run: |
-          wget https://github.com/higepon/mosh/releases/download/mosh-0.2.8-rc3/mosh-0.2.8-rc3.tar.gz
+          curl -OL https://github.com/higepon/mosh/releases/download/mosh-0.2.8-rc3/mosh-0.2.8-rc3.tar.gz
           tar zvxf mosh-0.2.8-rc3.tar.gz
           cd mosh-0.2.8-rc3
-          ./configure
-          make
+          ./configure --prefix=$HOME/opt/mosh
+          make -j$(sysctl -n hw.ncpu)
           make test
           sudo make install
+          echo "$HOME/opt/mosh/bin" >> $GITHUB_PATH
       - uses: actions/checkout@v3
       - name: Build and test the latest changes.
         run: |
           cd ${{ github.workspace }}
+          # Workaround to use homebrew bison: Pregenerate these
+          $HOMEBREW_PREFIX/opt/bison/bin/bison -p "number_yy" -d src/NumberReader.y -o src/NumberReader.tab.cpp
```

Comment:
This looks not very ideal. Can we just prioritize homebrew bison using PATH env?

Reply:
Let me take this back. Updating PATH is not a great idea.